visualizer/examples.js

```diff
@@ -205,11 +208,11 @@
 
   // Save the current contents of all examples to localStorage.
   function saveExamples() {
-    var elems = domUtil.$$('#exampleContainer ul li code');
-    var examples = elems.map(function(elem) {
-      return elem.textContent;
-    });
-    localStorage.setItem('examples', JSON.stringify(examples));
+    localStorage.setItem('examples', JSON.stringify(
+      Object.keys(exampleValues).map(function(key) {
```

Comment:
Can you just stick the result of the map in a local variable named `examples` like it was before? I think it reads more clearly that way.